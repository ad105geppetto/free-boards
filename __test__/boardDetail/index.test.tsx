import "@testing-library/jest-dom";
import { act, render } from "@testing-library/react";
import BoardDetailUI from "../../src/components/units/board/detail/BoardDetail.presenter";
import { propsMock } from "./index.mock";

describe("자유게시판 상세 페이지 Unit Test", () => {
  it("게시판 상세 페이지 스냅샷 테스트", async () => {
    await act(async () => {
      const result = render(<BoardDetailUI {...propsMock} />);
      expect(result.container).toMatchSnapshot();
    });
  });
});
