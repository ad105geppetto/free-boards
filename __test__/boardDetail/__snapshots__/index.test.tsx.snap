// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`자유게시판 상세 페이지 Unit Test 게시판 상세 페이지 스냅샷 테스트 1`] = `
<div>
  <div
    class="css-1ol1jm3"
  >
    <div
      class="css-ztv6jj"
    >
      <div
        class="css-2cj54t"
      >
        <a
          class="css-mq3wpp"
          href="/free-boards"
        >
          목록
        </a>
      </div>
      <section
        class="css-1sl02zy"
      >
        <h3
          class="css-xno0ws"
        >
          게시글 제목
        </h3>
        <div
          class="css-rb483s"
        >
          홍길동
        </div>
        <div
          class="css-npfqe2"
        >
          2023.04.01. 15:37
        </div>
        <div
          class="css-lz1axn"
        >
          게시글 내용
        </div>
      </section>
      <div
        class="css-1bzjq2p"
      >
        <button
          class="css-gn3lwt"
        >
          게시글 삭제하기
        </button>
        <button
          class="css-gn3lwt"
        >
          게시글 수정하기
        </button>
      </div>
    </div>
  </div>
</div>
`;
